aws-ssm-document/pom.xml

```diff
@@ -30,7 +30,7 @@
         <dependency>
             <groupId>software.amazon.awssdk</groupId>
             <artifactId>ssm</artifactId>
-            <version>2.10.66</version>
+            <version>[2.15.0, 3.0.0)</version>
```

Comment:
This will pick the latest aws sdk available. Not sure if this the the right way to go about it. I will let @jotompki comment on this.

Reply:
This could cause some issues if the ssm sdk is of a higher version than the sdk core specified in the aws-cloudformation-rpdk-java-plugin. I'm currently looking into solutions which would inherit properties from the aws-cloudformation-rpdk-java-plugin. It's best for now to just explicitly specify a version.

Reply:
updated aws sdk version to be the same as https://github.com/aws-cloudformation/cloudformation-cli-java-plugin/commit/75133adfd5b82c347bf31579b09e82750b47cde7